pandas/tests/io/test_parquet.py

```diff
@@ -486,3 +486,16 @@ def test_filter_row_groups(self, fp):
                           row_group_offsets=1)
             result = read_parquet(path, fp, filters=[('a', '==', 0)])
         assert len(result) == 1
+
+
+class TestIntegrationWithS3(Base):
+    def test_s3_roundtrip(self, df_compat, s3_resource, engine):
+        # GH #19134
+
+        if engine == 'pyarrow':
+            df_compat.to_parquet('s3://pandas-test/test.parquet', engine)
+
```

Comment:
don’t inherit Base nore use a class
make just a function
use the fixtures instead of engine directly

Reply:
@jreback how about now? Am I not using the fixture already? Not sure what you mean by that.